content/posts/0.27-release.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Pyodide 0.27 Release"
+date: 2024-09-27T11:00:55-04:00
+draft: true
+tags: ["announcement"]
+# author: "Me"
+author: ["Gyeongjae Choi"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    # image: "<image path/url>" # image path/url
+    # alt: "<alt text>" # alt text
+    # caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+Pyodide v0.27.0 is out. This release focuses on improving the long-term stability of Pyodide.
+
+# Welcome Agriya Khetarpal to the Pyodide team
+
+Agriya Khetarpal has joined Pyodide as the new maintainer.
+Agriya has been active in the Scientific Python area and is a contributor of Numpy, Scipy, and scikit-learn.
+He has already contributed significantly to strengthening Pyodide support in various Scientific Python packages.
+
+# Pyodide 0.27 Release
+
+## Build System Improvements
+
+### Decoupling pyodide-build from Pyodide runtime
+
+pyodide-build is a tool that builds Python packages for the WASM environment to run in Pyodide.
+Previously, the version of pyodide-build was strongly tied to the version of Pyodide,
+meaning that if you wanted to build a package against a specific version of Pyodide, you had to use a corresponding version of pyodide-build.
+The problem with this approach was that even if we improved the build system, you would have to wait a few months after Pyodide was released to use it.
+
+In this release, we have separated pyodide-build from the Pyodide runtime.
+This allows us to develop and release pyodide-build independently of Pyodide more quickly,
+and you can apply these improvements to your packages sooner.
+
+### New Packages Support
+
+This release includes about a dozen new packages, most notably PyArrow and DuckDB, which people have long requested.
+What's unique about these new packages is that they are not being built from the Pyodide repository,
+but are being built out-of-tree by the package maintainers using pyodide-build.
+
+Previously, Pyodide has built all packages directly from within the repository.
+This had been a significant limitation on long-term scalability,
+especially for large libraries like PyArrow, which take tens of minutes or more to build.
+Now, these packages are built in a separate repository and managed by the package maintainers.
+We expect this change will improve package stability and pave the way for more packages to be added to Pyodide in the long run.
```

Comment:
It might be good to include a link to https://github.com/pyodide/pyodide/issues/4918 here at the end, since it is an RFC issue and we would appreciate thoughts from other people.